source/_integrations/enelgrid.markdown

```diff
@@ -0,0 +1,63 @@
+---
+title: EnelGrid
+description: Integration for Enel Grid consumption data in Home Assistant
+stage: integration
+iot_class: cloud_polling
+integration_type: service
+dependencies:
+  - recorder
+codeowners:
+  - "@sathia-musso"
+ha_release: 2025.4
+ha_domain: enelgrid
+ha_integration_type: integration
+---
+
+The **EnelGrid** integration connects Home Assistant with **Enel Italia**, allowing users to import and monitor their electricity consumption data.
+
+## Features
+
+- Fetches hourly electricity consumption data
+- Tracks daily and cumulative monthly usage
+- Integrates with the Home Assistant Energy Dashboard
+- Automatically updates daily
+- Supports secure re-authentication if needed
+
+## Configuration
+
+During setup, you’ll be prompted to enter:
+
+- **Username**: Your Enel account email address
+- **Password**: Your Enel account password
+- **POD Number**: Found on your Enel electricity bill
+- **User Number**: Also found on your bill
+- **Price per kWh**: Estimated based on your billing rate (total cost ÷ total kWh consumed)
+
+All credentials are stored securely by Home Assistant’s configuration system.
+
```

Comment:
```suggestion
{% include integrations/config_flow.md %}

```
If the integration supports config flow, there a [reusable text block]({% include integrations/config_flow.md %}) you can use for the Configuration section.